tasks/setup-RedHat.yml

```diff
@@ -9,15 +9,41 @@
     nodejs_rhel_rpm_dir: "pub"
   when: nodejs_version == '0.10'
 
-- name: Import Nodesource RPM key.
+- name: Import Nodesource RPM key over https.
```

Comment:
Yeah. I guess this description is inaccurate. Would `Import Nodesource RPM key using the rpm_key module.` be better?